package.json

```diff
@@ -32,18 +32,18 @@
     "prebuild-install": "7.1.1"
   },
   "devDependencies": {
-    "chai": "^4.3.7",
+    "chai": "^4.3.10",
     "chai-string": "^1.5.0",
     "chalk": "^4.1.2",
     "clang-format": "^1.8.0",
     "dmd-clear": "^0.1.2",
-    "eslint": "^8.46.0",
+    "eslint": "^8.55.0",
     "eslint-config-prettier": "^8.10.0",
     "eslint-plugin-prettier": "^4.2.1",
     "jsdoc-to-markdown": "^8.0.0",
     "mocha": "^10.2.0",
-    "mongodb": "^5.7.0",
-    "node-gyp": "^9.4.0",
+    "mongodb": "^5.9.2",
```

Comment:
Missed this in kickoff I think, and this follows the AC, but why not bump to latest 6.x?

Reply:
Looking at where it's used, there's no reason not to, so we can update that here and in the ticket